R/bookmark-state.R

```diff
@@ -448,7 +445,7 @@ RestoreInputSet <- R6Class("RestoreInputSet",
 )
 
 
-restoreCtxStack <- Stack$new()
+restoreCtxStack <- fastmap::faststack()
```

Comment:
Would this possibly have issues in binaries if `fastmap` udpated it's c bindings?  

Before, the R6 object was _safe_.  Now, with info being offload to C in `fastmap`, is this a concern?

We could have it be a placeholder and update the value in `.onLoad()`

Ex:

```r
restoreCtxStack <- NULL

.onLoad <- function() {
  restoreCtxStack <- fastmap::faststack()
}
```

Reply:
Good catch -- `faststack()` doesn't call any C code and the object returned should be completely self-contained, but you're right that it's safer to create in `.onLoad`.